ros_ign_gazebo_demos/launch/depth_camera.launch.py

```diff
@@ -62,7 +63,7 @@ def generate_launch_description():
     # FIXME: need a SDF file (depth_camera.sdf) inside ros_ign_point_cloud
     return LaunchDescription([
         DeclareLaunchArgument(
-          'args',
+          'ign_args',
           #default_value=['depth_camera.sdf'],
```

Comment:
you need to define at least a default value
```suggestion
          default_value=[],
          #default_value=['depth_camera.sdf'],
```

Reply:
Ah thanks! This file is not working right now but it's good to take this out of the way.